x-pack/plugins/ml/public/jobs/new_job/simple/components/bucket_span_estimator/bucket_span_estimator_view.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import {
+  EuiButton,
+  EuiToolTip
+} from '@elastic/eui';
+
+export function BucketSpanEstimator({ buttonDisabled, buttonText, estimatorRunning, guessBucketSpan }) {
+  return (
+    <div className="bucket-span-estimator">
+      <EuiToolTip
+        content="Experimental feature for estimating bucket span."
```

Comment:
I wonder if this warning about it being an experimental feature is still needed?

Reply:
We are adding in k-api tests shortly. I'm not sure how far we'll get in terms of depth of coverage in this release,  so let's keep as experimental till we have a stronger set of data scenario tests.